consensus/src/experimental/commit_phase.rs

```diff
@@ -141,9 +148,10 @@ pub struct CommitPhase {
     author: Author,
     back_pressure: Arc<AtomicU64>,
     network_sender: NetworkSender,
-    timeout_event_tx: Sender<CommitVote>,
-    timeout_event_rx: Receiver<CommitVote>,
+    timeout_event_tx: Sender<CommitPhaseTimeoutEvent>,
+    timeout_event_rx: Receiver<CommitPhaseTimeoutEvent>,
     reset_event_rx: Receiver<oneshot::Sender<ResetAck>>,
+    in_epoch: bool,
```

Comment:
end_epoch?

Reply:
it means whether the commit phase is in the epoch (the contrary to end_epoch).

Reply:
`in_epoch` is a little confusing, since it should always be in an epoch. we can flip the bool and name is as `epoch_end` or simply `stop`

Reply:
okay I'll change it later